Doc/library/os.rst

```diff
@@ -1659,10 +1659,11 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
       Added support for pipes on Windows.
 
 
-.. function:: splice(src, dst, count, offset_src=None, offset_dst=None)
+.. function:: splice(src, dst, count, offset_src=None, offset_dst=None, flags=0)
 
    Transfer *count* bytes from file descriptor *src*, starting from offset
-   *offset_src*, to file descriptor *dst*, starting from offset *offset_dst*.
+   *offset_src*, to file descriptor *dst*, starting from offset *offset_dst*,
+   which can use *flags* to modify the semantics of the call.
```

Comment:
This is very vague! What are the allowed flags, what behaviour changes are effected, are they bitwiseable, etc?

Reply:
I've added some descriptions from the man pages with a link to it. Let me know if the wording is not good.